### PR TITLE
truncate to 2 avatars on mobile with remaining indicator

### LINKED ISSUE #2096: only show 2 avatars in avatargroup on mobile
image

### PR DESCRIPTION
**Pull request by Mayowa-Ojo**
## What does this PR do?

Allows the [`AvatarGroup`](https://github.com/calcom/cal.com/blob/main/apps/web/components/ui/AvatarGroup.tsx) component to display `2` avatars on mobile along with an indicator showing how much more are hidden. 

Fixes #2096 

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How should this be tested?

- Run the app and open a page that renders the `AvatarGroup` component (e.g team availability) in mobile view. 

## Checklist

- I haven't performed a self-review of my own code and corrected any misspellings
- I haven't checked if my PR needs changes to the documentation
- I haven't added tests that prove my fix is effective or that my feature works

### CODE REVIEW

**Comment by PeerRich** [comment]
thank you @Mayowa-Ojo! Will test today

**Comment by PeerRich** [comment]
+0 is not really useful when it's only two peope haha

image

File: apps/web/components/ui/AvatarGroup.tsx [hunk 1]
```diff
@@ -18,23 +18,50 @@ export type AvatarGroupProps = {
...
+                  size={props.size}
+                />
+              </li>
+            );
+          }
+        })}
+        <li className="-mr-2 inline-block">
+          <button className="cursor-default">
+            <span className="flex h-9 w-9 items-center justify-center rounded-full border-2 border-white bg-gray-100 dark:border-gray-900">
+              <p className="text-gray-800">+{props.items.length - 2}</p>
```
Comment (sean-brydon) [inline apps/web/components/ui/AvatarGroup.tsx:42]:
```suggestion
              {props.items.length-2 >0 && <p className="text-gray-800">+{props.items.length - 2}</p>}
```

Leaving this as a suggestion as I don't have a laptop right now

Comment (sean-brydon) [reply apps/web/components/ui/AvatarGroup.tsx:42]: Looking at this again we should be able to just remove the whole list element

**Comment by PeerRich** [comment]
@sean-brydon can you take this over?

**Comment by PeerRich** [comment]
actually, I will rebuild the entire UI so it doesn't need truncating!

**Comment by PeerRich** [comment]
nevertheless thank you @Mayowa-Ojo for the PR!

**Comment by PeerRich** [comment]
image

fixed

**Pull request closed without merging**